news-bundle/src/Resources/contao/dca/tl_news_feed.php

```diff
@@ -440,14 +455,14 @@ public function generateFeed()
 			return;
 		}
 
-		$this->import('Contao\News', 'News');
+		$this->import('News', 'News');
```

Comment:
Shouldn't this be changed to

```php
$this->import('News')
````
?

Also.. does this actually work? May be it needs to be

```php
$this->import(News::class, 'News')
```

? (I never use `$this->import` myself, so I am just unsure about it)

Reply:
You are right, it seems that I have missed some occurrences. Fixed in #1863.